public class ThrowsArithmeticExc {

    public static void main(String[] args) {
        int a = 45;
        int b = 0;
        try {
            div(a, b);
        } catch (ArithmeticException e) {
            System.out.println("Arithmetic Exception");
        }
    }

    static void div(int x, int y) throws ArithmeticException {
        int c = x / y;
        System.out.println("div : " + c);
    }
}
